src/test/analyzer_test.ts

```diff
@@ -12,15 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-'use strict';
+/// <reference path="../../node_modules/@types/mocha/index.d.ts" />
```

Comment:
Weird, because in other projects I don't have a reference to it at all. What tsc are you on?